lms/static/scripts/frontend_apps/components/Table.js

```diff
@@ -24,8 +29,34 @@ function nextItem(items, currentItem, step) {
   return items[index + step];
 }
 
+/**
+ * @typedef TableColumn
+ * @prop {string} label - Header label for the column
+ * @prop {string} className - Additional classes for the column's `<th>` element
+ */
+
+/**
+ * @template Item
```

Comment:
Another super-trivial thing: can `@template` go beneath `@typedef` here? I tend to scan blocks and try to see the top line for the type being defined.

Reply:
> Another super-trivial thing: can @template go beneath @typedef here? 

No - it seems TS complains if you do that.